### ISSUE TITLE
Add overlays for video

### ISSUE THREAD

**Issue by vanatka** [viromedia/virocore#46]
Hello Dear Viro support team, 

Lets imagine I left an object on scene, and also above the scene I'm showing some frame, for example photo frame as ImageView above Surface view.
Now I want capture video, is there any way add overlay to the video? 
Should it be added as object on scene or there is another way? 

Many thanks, kind regards, Ivan

**Comment by dthian** [comment]
Hey @vanatka, a few quick questions:
- where would you want to overlay the captured video again? 
- Would that video feed be in the imageView above the surfaceView? 
- Would this a video recording that you then want to play back in the feed?

**Comment by vanatka** [comment]
@dthian , oh seems, my question was not clear, let me rephrase it.
btw: I'm talking about AR.
as far as know, using your Viro SDK I can capture videos, 

So please look at picture below, here is ViroView with 3d cube's model.
Above ViroView there is FrameLayout where is located PNG frame, borders of frame are black, the background of PNG image is transparent, sorry for my painting skills..

![2018-03-19 11 51 03](https://user-images.githubusercontent.com/2757506/37606489-f5ea3280-2b6b-11e8-81ad-712268884027.jpg)

Is there any way include into the video, the frame from FrameLayout, to put it above video stream.
I will save that video on SD Card, just regular case, without any additional magic, and then will make playback in some standard media player.

**Comment by dthian** [comment]
Hi @vanatka, thanks for your detailed pictures and explaining the situation! 👍 

Unfortunately ViroCore only records what is being rendered within the ViroView, it does not record what is being displayed in other Android views. This is done so that developers can then just use Viro's video recording for saving render-specific frames, else they can simply use Android APIs to save the entire rendered activity.

As such, it sounds like you are attempting to do a full screen capture of Android views in general - this should already be [supported by native Android APIs](https://developer.android.com/about/versions/android-5.0.html#ScreenCapture). [Here](https://github.com/googlesamples/android-ScreenCapture/blob/master/Application/src/main/java/com/example/android/screencapture/ScreenCaptureFragment.java) is another example from google on how it can be implemented. This is probably the easiest and most straight forward solution.
